src/index.js

```diff
@@ -200,6 +192,62 @@ class FloodSub extends EventEmitter {
   }
 
   /**
+   * Mounts the floodsub protocol onto the libp2p node and sends our
+   * subscriptions to every peer conneceted
+   *
+   * @param {Function} callback
+   * @returns {undefined}
+   *
+   */
+  start (callback) {
```

Comment:
I believe the improving init -- https://github.com/ipfs/js-ipfs/issues/556 -- discussion comes from the annoyance of having 3 steps to start js-ipfs (init, load, goOnline), while in bitswap, floodsub, libp2p it is really just one